### DISCUSSION TITLE
Unable to register_character using the bubble style

### DISCUSSION THREAD

**Discussion by meanbob1029** [dialogic-godot/dialogic#2070] [Q&A] [answered]
i was following along with a tutorial for Godot (new to Godot) in the tutorial he uses `Layout.register_character(BobRef, $Villager_Bob)` to connect the Bubble text to the character, if i leave out this line of code everything works but the chat bubbles are just in the middle not over the characters heads.
the game is crashing everytime this is called stating the function regisiter_character is invalid. 
i have looked at the documentation and my code seems to be inline with that it is telling me to do. 
i have also tried different versions of the plugin. i am using Godot 4.2.1 stable
and my last try i used  v2-alpha-13 version of dialogic
am i doing something wrong? 

![image_2024-02-05_002049833](https://github.com/dialogic-godot/dialogic/assets/24641274/efb3b0a9-88db-4c3e-a879-11e5baee2fa3)

**Comment by Jowan-Spooner** [comment]
Something is certainly going wrong, as your layout should use the text-bubble-base. Can you show me the style editor where you've setup the textbubble style?

**Comment by ZR369000** [comment, marked as answer]
I also encountered the same problem and it took me a whole day to realize that it was because I had not set the default style.

**Reply by CakeVR** [reply]
This is correct. If there is no style set in the Dialogic Style editor **or** the **default** style is **not** `Textbubble Style`, the error will appear as above seen.

How to mark a style as default? Select a style from the style list in the Dialogic Style editor, then click the button with a star on it.

**Reply by SEED-exe** [reply]
but when you click on the star nothing happens..
